com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/flowable/FlowableFacade.java

```diff
@@ -197,6 +197,20 @@ public Execution getProcessExecution(String processInstanceId) {
             .orElse(null);
     }
 
+    public List<HistoricVariableInstance> findHistoricVariablesForProcess(String processInstanceId) {
```

Comment:
You only use this method to find one specific variable by its name. Delete this method and use `getHistoricVariableInstance(String processInstanceId, String variableName)` instead.